std-bits/database/src/main/java/org/enso/database/JDBCUtils.java

```diff
@@ -12,6 +12,17 @@
 import org.enso.polyglot.common_utils.Core_Date_Utils;
 
 public class JDBCUtils {
```

Comment:
Missing `private` constructor. The default constructor is `public` hence people could create instances of this _utility methods_ only class, which makes little sense.